samples/cellular/coap_client/boards/nrf7002dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configuration file for nRF7002 DK
+# This file is merged with prj.conf in the application folder, and options
+# set here will take precedence if they are present in both files.
+
+# General
+CONFIG_POSIX_CLOCK=y
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_LOG_BUFFER_SIZE=4096
+CONFIG_HEAP_MEM_POOL_SIZE=153600
```

Comment:
Why do we need 153kb of heap?

Reply:
This seems to be necessary for the WiFi stack. Not sure about the exact size requirements, I've taken this value from `aws_iot` sample.

Reply:
that is indeed due to extended heap usage of the wifi stack